docs/tips.md

```diff
@@ -96,6 +137,26 @@ return {
 }
 ```
 
+Add the following instead if you're also using `zoxide`:
```

Comment:
Could you explain its usage?

Reply:
Apologies for my late reply. I didn't test it enough and didn't realize that it doesn't work. ref: https://github.com/sxyazi/yazi/issues/1087. I have removed this part. Thank you!